haskell/nettest/Main.hs

```diff
@@ -10,29 +10,34 @@ import qualified Unsafe (fromJust)
 import qualified Indigo.Contracts.Transferlist.External as External
 import qualified Indigo.Contracts.Transferlist.Internal as Internal
 import Lorentz (ToT, compileLorentzContract, toVal)
+import Lorentz.Contracts.Stablecoin (stablecoinPath)
 import Lorentz.Contracts.Test.Common
 import Michelson.Runtime (parseExpandContract, prepareContract)
 import Michelson.Typed (untypeValue)
 import Michelson.Typed.Convert (convertContract)
 import Morley.Nettest
-import Nettest (scNettestScenario)
+import Stablecoin.Client.Cleveland.Caps (runStablecoinClient)
 import Tezos.Address
 import Util.IO
 import Util.Named
 
+import Nettest (scNettestScenario)
+import StablecoinClientTest (stablecoinClientScenario)
+
 externalTransferlistContractPath :: FilePath
 externalTransferlistContractPath = "test/resources/transferlist.tz"
 
 main :: IO ()
 main = do
-  let clientParser = clientConfigParser . pure $ Just "nettest.Stablecoin"
+  let aliasPrefix = "nettest.Stablecoin"
+  let clientParser = clientConfigParser . pure $ Just aliasPrefix
   parsedConfig <- execParser $
     parserInfo
       (#usage .! mempty)
       (#description .! "Stablecoin nettest scenarioWithInternalTransferlist")
       (#header .! "ManagedLedger nettest")
       (#parser .! clientParser)
-  stablecoinContract <- prepareContract $ Just "test/resources/stablecoin.tz"
+  stablecoinContract <- prepareContract $ Just stablecoinPath
```

Comment:
Since the contract is now embedded into source code, don't we want to use the embedded version here? I. e. use `parseStablecoinContract`. I know they are supposed to be the same, but in theory one may compile code from one folder and run tests from another folder (even on another machine) in which case `stablecoin-client` will use not the same contract as some tests. Also if we use `parseStablecoinContract` here, it will be easier to run these tests because we won't have to make sure that `"test/resources/stablecoin.tz"` is available when we **run** the tests.

Reply:
Good idea! Done, and changed this in a couple of other places as well.